addons/cni-canal/canal.yaml

```
@@ -4341,7 +4483,7 @@ spec:
             privileged: true
           resources:
             requests:
-              cpu: {{ default .Params.RequestsCPU "250m" }}
+              cpu: 250m
```

Please revert this change. 
```suggestion
             cpu: {{ default .Params.RequestsCPU "250m" }}
```

---

Fixed.